fugit2.nvim-scm-1.rockspec

```diff
@@ -0,0 +1,37 @@
+package	= 'fugit2.nvim'
+
+version	= 'scm-1'
+
+rockspec_format = '3.0'
+
+source	= {
+	url	= 'git://github.com/SuperBo/fugit2.nvim.git'
+}
+
+description	= {
+	summary	= 'Git plugin for Neovim (based on libgit2)',
+	homepage	= 'https://github.com/SuperBo/fugit2.nvim',
+	license	= 'MIT',
+}
+
+dependencies = {
+	'lua >= 5.1',
```

Comment:
Can we specify lua version to luajit because the implementation is based on luajit ffi, and don't work with normal lua.

Reply:
This is just an API version spec. Luarocks won't actually download and install lua.
luajit targets the lua 5.1 API, and neovim plugins generally specify `lua >= 5.1` in their dependencies list, which influences the install directory.
In most cases, neovim comes bundled with luajit. Luarocks has no way of checking the lua version neovim is bundled with.